api/app/controllers/spree/api/checkouts_controller.rb

```diff
@@ -42,7 +42,7 @@ def complete
       def update
         authorize! :update, @order, order_token
 
-        if OrderUpdateAttributes.new(@order, update_params, request_env: request.headers.env).apply
+        if Spree::Config.order_update_attributes_class.new(@order, update_params, request_env: request.headers.env).apply
```

Comment:
We've been trying to standardize to `#call` for these injected classes' interfaces. We could alias it to `#apply` so we don't break backward compatibility.

Reply:
Done